modules/ROOT/nav.adoc

```diff
@@ -17,6 +17,7 @@
 * xref:api-development-studio.adoc[API Development in Studio]
 
 * xref:create-api-specification-studio.adoc[Create an API Project in Studio]
+* xref:test-specification-api-console.adoc[Test your API Specification Using the API Console]
```

Comment:
```suggestion
* xref:test-specification-api-console.adoc[Test Your API Specification Using the API Console]
```
Although there are four main title capitalization styles, MuleSoft Tech Docs follows Chicago Manual of Style: 
Capitalize the first and the last word.
Capitalize nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions.
Lowercase articles (a, an, the), coordinating conjunctions, and prepositions of four or fewer letters.
Lowercase the ‘to’ in an infinitive

When in doubt, use Capitalize My Title (choose the Chicago tab).